packages/fa-icon-chooser/dev/runtime.js

```diff
@@ -1,105 +1,99 @@
 // This dev-only module isn't processed by the bundler like the others,
 // so we can't use a node env var to set this. Just hardcode it in one
 // place at the top.
-const API_URL = "https://api.fontawesome.com";
+const API_URL = 'https://api-epic.fontawesome.com';
 
 const FaIconChooserDevExports = (function () {
   let showingIconChooser = false;
   let localConfig = undefined;
-  const localDevMissingMsg =
-    "DEV: your local dev config in local.json is required but has not yet been loaded.";
+  const localDevMissingMsg = 'DEV: your local dev config in local.json is required but has not yet been loaded.';
```

Comment:
All changes from this point down, in the file is code formatting fixups... no code changes.